app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics");
+
+
+app.use((err, req, res, next) => {
+  if (err.message === "Topics not found") {
+    res.status(404).send({ message: "Topics not found" });
+  } else {
+    res.status(500).send({ message: "Internal server error" });
+  }
+});
+
```

Comment:
You should definitely be putting your error handling into app.js as the LAST bit of middleware that happens not first